server/graphql/common/expenses.ts

```diff
@@ -342,6 +356,31 @@ export const rejectExpense = async (
   return updatedExpense;
 };
 
+export const markExpenseAsSpam = async (
+  req: express.Request,
+  expense: typeof models.Expense,
+): Promise<typeof models.Expense> => {
+  if (expense.status === expenseStatus.SPAM) {
+    return expense;
+  } else if (!(await canMarkAsSpam(req, expense))) {
+    throw new Forbidden();
+  }
+
+  const updatedExpense = await expense.update({ status: expenseStatus.SPAM, lastEditedById: req.remoteUser.id });
+
+  // Limit the user so they can't submit expenses in the future
+  const submittedByUser = await updatedExpense.getSubmitterUser();
+  await submittedByUser.limitFeature(FEATURE.USE_EXPENSES);
+
+  // We create the activity as a good practice but there is no email sent right now
+  const activity = await expense.createActivity(activities.COLLECTIVE_EXPENSE_MARKED_AS_SPAM, req.remoteUser);
```

Comment:
We may want to add this activity in https://github.com/opencollective/opencollective-api/blob/1d321f312479e98cfd99dcfd34d20223fe31fdeb/server/graphql/loaders/expenses.ts#L61 to surface it in the interface